components/style/custom_properties.rs

```diff
@@ -131,7 +130,7 @@ impl ToCss for SpecifiedValue {
 ///
 /// The variable values are guaranteed to not have references to other
 /// properties.
-pub type CustomPropertiesMap = OrderedMap<Name, Arc<VariableValue>>;
+pub type CustomPropertiesMap = IndexMap<Name, Arc<VariableValue>>;
```

Comment:
This needs to use `BuildHasherDefault<PrecomputedHasher>` as the third hash parameter, in order to avoid hashing keys over and over, since `Name` is an interned string and has a precomputed hash already.

Some perf measurements with this change would be good, but I can take care of them once it's done.